docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -193,31 +255,216 @@ You should receive a response indicating there were no errors.
 ===============
 
 [discrete]
-[[qs-search-data]]
-=== Search data
+[[getting-started-mappings-and-data-types]]
+=== Step 3: Define mappings and data types
+
+<<elasticsearch-intro-documents-fields-mappings,Mappings>> define how data is stored and indexed in {es}, like a schema in a relational database.
+
+[discrete]
+[[getting-started-dynamic-mapping]]
+==== Use dynamic mapping
+
+When using dynamic mapping, {es} automatically creates mappings for new fields by default.
+The documents we've added so far have used dynamic mapping, because we didn't specify a mapping when creating the index.
+
+Add a new document to the `books` index with a field that doesn't exist, to see how dynamic mapping works.
+
+[source,console]
+----
+POST /books/_doc
+{
+  "name": "The Great Gatsby",
+  "author": "F. Scott Fitzgerald",
+  "release_date": "1925-04-10",
+  "page_count": 180,
+  "new_field": "I'm a new field added dynamically" <1>
```

Comment:
in my original note, I meant we could fancy it up a little to make it feel more like a book field. if you don't feel like changing the example field, then I don't think the annotation is necessary.

also: I say this in another comment, but is the field added dynamically? 

```suggestion
  "language": "EN" <1>
```

Reply:
ah yes thanks